### PR TITLE
docs: observability Datadog integration

### PR DESCRIPTION
**Pull request by sissbruecker**

### CODE REVIEW

File: .github/styles/Vocab/Docs/accept.txt [hunk 1]
```diff
@@ -94,6 +97,7 @@ Roboto
 (s|S|des|Des)erializ(e|er|ers|ed|ing|able)
 [sS]erverless
 [sS]ervlets?
+[sS]ignup
```
Comment (jouni) [inline .github/styles/Vocab/Docs/accept.txt:100]:
```suggestion
```

File: articles/tools/observability/integrations/datadog.asciidoc [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+https://www.datadoghq.com/[Datadog] is a commercial service for collecting and viewing telemetry data.
+
+The Datadog integration supports metrics and traces. Logging integration is currently limited to the Logback logging framework.
+
+== Setup
+
+=== Account Setup
+Start by setting up a Datadog account. Open the https://www.datadoghq.com/[Datadog website] and click on `Free Trial`. Select the appropriate region for storing data, fill out the remaining details and follow the instructions.
+
+In order to complete the signup you *must* install one of their agents and have it send some data.
```
Comment (jouni) [inline articles/tools/observability/integrations/datadog.asciidoc:19]:
```suggestion
In order to complete the sign-up process you *must* install one of their agents and have it send some data.
```

File: articles/tools/observability/integrations/datadog.asciidoc [hunk 3]
```diff
@@ -0,0 +1,269 @@
...
+The agent is not required for the Vaadin Observability integration, and can be removed immediately afterwards.
+The last signup step provides instructions for installing the agent. An easy way to run the agent once is to use the Docker option, which allows to create a container, send some data, and then stop and remove the container immediately afterwards.
```
Comment (jouni) [inline articles/tools/observability/integrations/datadog.asciidoc:21]:
```suggestion
The last sign-up step provides instructions for installing the agent. An easy way to run the agent once is to use the Docker option, which allows to create a container, send some data, and then stop and remove the container immediately afterwards.
```

**Review by caalador — COMMENTED** [review]
Missing link in index.asciidoc for dataDog

**Comment by sissbruecker** [comment]
> Missing link in index.asciidoc for dataDog

Added a link to the index page.

**Review by caalador — APPROVED** [review]

**Pull request merged**